x-pack/plugins/apm/server/lib/services/get_service_error_groups/get_service_error_group_detailed_statistics.ts

```diff
@@ -159,15 +159,15 @@ export async function getServiceErrorGroupPeriods({
     previousPeriodPromise,
   ]);
 
-  const firtCurrentPeriod = currentPeriod.length ? currentPeriod[0] : undefined;
+  const firstCurrentPeriod = currentPeriod?.[0];
```

Comment:
@cauemarcondes I think you wrote this originally, so just want to all out how it's possible to avoid the ternary using optional chaining operator (`?.`)

Reply:
What do you mean @sqren , this basically returns undefined if `currentPeriod` is undefined, so there's no need to do a ternary in here.

Reply:
Agree, which is why I removed the ternary. I thought you added it originally.